R/metapredict_predict.R

```diff
@@ -171,14 +185,14 @@ makeCoefDf = function(fitResult, lambda, decreasing=TRUE, classLevels=NA) {
 
     for (ii in 1:length(coefResult)) {
       colnames(coefResultNonzero[[ii]])[2] = names(coefResult)[ii]}
-    coefDf = Reduce(function(x, y) merge(x, y, by='geneId', all=TRUE), coefResultNonzero)
-    idx = do.call(order, c(coefDf[,2:ncol(coefDf)], list(decreasing=decreasing)))
+    coefDf = Reduce(function(x, y) merge(x, y, by = 'geneId', all = TRUE), coefResultNonzero)
+    idx = do.call(order, c(coefDf[,2:ncol(coefDf)], list(decreasing = decreasing)))
     coefDf = coefDf[idx,]
     coefDf[is.na(coefDf)] = 0
 
   } else {
-    coefDf = data.frame(names(coefResult[(coefResult@i)+1,]), coefResult[(coefResult@i)+1,], stringsAsFactors=FALSE)
+    coefDf = data.frame(names(coefResult[(coefResult@i)+1,]), coefResult[(coefResult@i)+1,], stringsAsFactors = FALSE)
```

Comment:
why does `coefDf` need to be a data.frame, especially if the rownames later get set to NULL?